### ISSUE TITLE
[vscoq2] Goal display should not have a bunch of Ppcmd in it

### ISSUE THREAD

**Issue by JasonGross** [coq-community/vscoq#572]
Not sure how this happened / how to reproduce, but figured it's worth reporting
```
Goal 1

d_vocab, n_heads, d_model, d_head : Ppcmd_tagconstr.variablePpcmd_stringShapeType
n_ctx : Ppcmd_tagconstr.variablePpcmd_stringN
r : Ppcmd_tagconstr.variablePpcmd_stringRank
batch : Ppcmd_boxPp_hovbox2Ppcmd_gluePpcmd_tagconstr.variablePpcmd_stringShapePpcmd_gluePpcmd_print_break10Ppcmd_tagconstr.variablePpcmd_stringr
pos : Ppcmd_tagconstr.variablePpcmd_stringShapeType
normalization_type : Ppcmd_boxPp_hovbox2Ppcmd_gluePpcmd_tagconstr.variablePpcmd_stringwith_defaultPpcmd_gluePpcmd_gluePpcmd_print_break10Ppcmd_string"normalization_type"Ppcmd_print_break10Ppcmd_boxPp_hovbox1Ppcmd_gluePpcmd_string(Ppcmd_boxPp_hovbox2Ppcmd_gluePpcmd_tagconstr.variablePpcmd_stringSomePpcmd_gluePpcmd_print_break10Ppcmd_tagconstr.variablePpcmd_stringLNPpcmd_string)
A : Ppcmd_tagconstr.typePpcmd_stringType
zeroA : Ppcmd_boxPp_hovbox2Ppcmd_gluePpcmd_tagconstr.variablePpcmd_stringhas_zeroPpcmd_gluePpcmd_print_break10Ppcmd_tagconstr.variablePpcmd_stringA
coerZ : Ppcmd_boxPp_hovbox2Ppcmd_gluePpcmd_tagconstr.variablePpcmd_stringhas_coerPpcmd_gluePpcmd_gluePpcmd_print_break10Ppcmd_tagconstr.variablePpcmd_stringZPpcmd_print_break10Ppcmd_tagconstr.variablePpcmd_stringA
addA : Ppcmd_boxPp_hovbox2Ppcmd_gluePpcmd_tagconstr.variablePpcmd_stringhas_addPpcmd_gluePpcmd_print_break10Ppcmd_tagconstr.variablePpcmd_stringA
subA : Ppcmd_boxPp_hovbox2Ppcmd_gluePpcmd_tagconstr.variablePpcmd_stringhas_subPpcmd_gluePpcmd_print_break10Ppcmd_tagconstr.variablePpcmd_stringA
mulA : Ppcmd_boxPp_hovbox2Ppcmd_gluePpcmd_tagconstr.variablePpcmd_stringhas_mulPpcmd_gluePpcmd_print_break10Ppcmd_tagconstr.variablePpcmd_stringA
divA : Ppcmd_boxPp_hovbox2Ppcmd_gluePpcmd_tagconstr.variablePpcmd_stringhas_divPpcmd_gluePpcmd_print_break10Ppcmd_tagconstr.variablePpcmd_stringA
sqrtA : Ppcmd_boxPp_hovbox2Ppcmd_gluePpcmd_tagconstr.variablePpcmd_stringhas_sqrtPpcmd_gluePpcmd_print_break10Ppcmd_tagconstr.variablePpcmd_stringA
expA : Ppcmd_boxPp_hovbox2Ppcmd_gluePpcmd_tagconstr.variablePpcmd_stringhas_expPpcmd_gluePpcmd_print_break10Ppcmd_tagconstr.variablePpcmd_stringA
eps : Ppcmd_boxPp_hovbox2Ppcmd_gluePpcmd_tagconstr.variablePpcmd_stringwith_defaultPpcmd_gluePpcmd_gluePpcmd_print_break10Ppcmd_string"use_checkpoint"Ppcmd_print_break10Ppcmd_tagconstr.variablePpcmd_stringtrue
use_checkpoint : Ppcmd_tagconstr.variablePpcmd_stringA
W_E : Ppcmd_boxPp_hovbox2Ppcmd_gluePpcmd_tagconstr.variablePpcmd_stringtensorPpcmd_gluePpcmd_gluePpcmd_print_break10Ppcmd_tagconstr.notationPpcmd_string[Ppcmd_boxPp_hovbox0Ppcmd_gluePpcmd_tagconstr.variablePpcmd_stringd_vocabPpcmd_tagconstr.notationPpcmd_string;Ppcmd_print_break10Ppcmd_tagconstr.variablePpcmd_stringd_modelPpcmd_tagconstr.notationPpcmd_string]Ppcmd_print_break10Ppcmd_tagconstr.variablePpcmd_stringA
W_pos : Ppcmd_boxPp_hovbox2Ppcmd_gluePpcmd_tagconstr.variablePpcmd_stringtensorPpcmd_gluePpcmd_gluePpcmd_print_break10Ppcmd_tagconstr.notationPpcmd_string[Ppcmd_boxPp_hovbox0Ppcmd_gluePpcmd_tagconstr.variablePpcmd_stringn_ctxPpcmd_tagconstr.notationPpcmd_string;Ppcmd_print_break10Ppcmd_tagconstr.variablePpcmd_stringd_modelPpcmd_tagconstr.notationPpcmd_string]Ppcmd_print_break10Ppcmd_tagconstr.variablePpcmd_stringA
blocks_params : Ppcmd_boxPp_hovbox2Ppcmd_gluePpcmd_tagconstr.variablePpcmd_stringlistPpcmd_gluePpcmd_print_break10Ppcmd_boxPp_hovbox1Ppcmd_gluePpcmd_string(Ppcmd_boxPp_hovbox0Ppcmd_gluePpcmd_boxPp_hovbox0Ppcmd_gluePpcmd_boxPp_hovbox0Ppcmd_gluePpcmd_boxPp_hovbox0Ppcmd_gluePpcmd_boxPp_hovbox0Ppcmd_gluePpcmd_boxPp_hovbox0Ppcmd_gluePpcmd_boxPp_hovbox0Ppcmd_gluePpcmd_boxPp_hovbox0Ppcmd_gluePpcmd_boxPp_hovbox0Ppcmd_gluePpcmd_boxPp_hovbox2Ppcmd_gluePpcmd_tagconstr.variablePpcmd_stringtensorPpcmd_gluePpcmd_gluePpcmd_print_break10Ppcmd_tagconstr.notationPpcmd_string[Ppcmd_boxPp_hovbox0Ppcmd_gluePpcmd_tagconstr.variablePpcmd_stringn_headsPpcmd_tagconstr.notationPpcmd_string;Ppcmd_print_break10Ppcmd_gluePpcmd_gluePpcmd_tagconstr.variablePpcmd_stringd_modelPpcmd_tagconstr.notationPpcmd_string;Ppcmd_print_break10Ppcmd_tagconstr.variablePpcmd_stringd_headPpcmd_tagconstr.notationPpcmd_string]Ppcmd_print_break10Ppcmd_tagconstr.variablePpcmd_stringAPpcmd_tagconstr.notationPpcmd_string *Ppcmd_print_break10Ppcmd_boxPp_hovbox2Ppcmd_gluePpcmd_tagconstr.variablePpcmd_stringtensorPpcmd_gluePpcmd_gluePpcmd_print_break10Ppcmd_tagconstr.notationPpcmd_string[Ppcmd_boxPp_hovbox0Ppcmd_gluePpcmd_tagconstr.variablePpcmd_stringn_headsPpcmd_tagconstr.notationPpcmd_string;Ppcmd_print_break10Ppcmd_gluePpcmd_gluePpcmd_tagconstr.variablePpcmd_stringd_modelPpcmd_tagconstr.notationPpcmd_string;Ppcmd_print_break10Ppcmd_tagconstr.variablePpcmd_stringd_headPpcmd_tagconstr.notationPpcmd_string]Ppcmd_print_break10Ppcmd_tagconstr.variablePpcmd_stringAPpcmd_tagconstr.notationPpcmd_string *Ppcmd_print_break10Ppcmd_boxPp_hovbox2Ppcmd_gluePpcmd_tagconstr.variablePpcmd_stringtensorPpcmd_gluePpcmd_gluePpcmd_print_break10Ppcmd_tagconstr.notationPpcmd_string[Ppcmd_boxPp_hovbox0Ppcmd_gluePpcmd_tagconstr.variablePpcmd_stringn_headsPpcmd_tagconstr.notationPpcmd_string;Ppcmd_print_break10Ppcmd_gluePpcmd_gluePpcmd_tagconstr.variablePpcmd_stringd_modelPpcmd_tagconstr.notationPpcmd_string;Ppcmd_print_break10Ppcmd_tagconstr.variablePpcmd_stringd_headPpcmd_tagconstr.notationPpcmd_string]Ppcmd_print_break10Ppcmd_tagconstr.variablePpcmd_stringAPpcmd_tagconstr.notationPpcmd_string *Ppcmd_print_break10Ppcmd_boxPp_hovbox2Ppcmd_gluePpcmd_tagconstr.variablePpcmd_stringtensorPpcmd_gluePpcmd_gluePpcmd_print_break10Ppcmd_tagconstr.notationPpcmd_string[Ppcmd_boxPp_hovbox0Ppcmd_gluePpcmd_tagconstr.variablePpcmd_stringn_headsPpcmd_tagconstr.notationPpcmd_string;Ppcmd_print_break10Ppcmd_gluePpcmd_gluePpcmd_tagconstr.variablePpcmd_stringd_modelPpcmd_tagconstr.notationPpcmd_string;Ppcmd_print_break10Ppcmd_tagconstr.variablePpcmd_stringd_headPpcmd_tagconstr.notationPpcmd_string]Ppcmd_print_break10Ppcmd_tagconstr.variablePpcmd_stringAPpcmd_tagconstr.notationPpcmd_string *Ppcmd_print_break10Ppcmd_boxPp_hovbox2Ppcmd_gluePpcmd_tagconstr.variablePpcmd_stringtensorPpcmd_gluePpcmd_gluePpcmd_print_break10Ppcmd_tagconstr.notationPpcmd_string[Ppcmd_boxPp_hovbox0Ppcmd_gluePpcmd_tagconstr.variablePpcmd_stringn_headsPpcmd_tagconstr.notationPpcmd_string;Ppcmd_print_break10Ppcmd_tagconstr.variablePpcmd_stringd_headPpcmd_tagconstr.notationPpcmd_string]Ppcmd_print_break10Ppcmd_tagconstr.variablePpcmd_stringAPpcmd_tagconstr.notationPpcmd_string *Ppcmd_print_break10Ppcmd_boxPp_hovbox2Ppcmd_gluePpcmd_tagconstr.variablePpcmd_stringtensorPpcmd_gluePpcmd_gluePpcmd_print_break10Ppcmd_tagconstr.notationPpcmd_string[Ppcmd_boxPp_hovbox0Ppcmd_gluePpcmd_tagconstr.variablePpcmd_stringn_headsPpcmd_tagconstr.notationPpcmd_string;Ppcmd_print_break10Ppcmd_tagconstr.variablePpcmd_stringd_headPpcmd_tagconstr.notationPpcmd_string]Ppcmd_print_break10Ppcmd_tagconstr.variablePpcmd_stringAPpcmd_tagconstr.notationPpcmd_string *Ppcmd_print_break10Ppcmd_boxPp_hovbox2Ppcmd_gluePpcmd_tagconstr.variablePpcmd_stringtensorPpcmd_gluePpcmd_gluePpcmd_print_break10Ppcmd_tagconstr.notationPpcmd_string[Ppcmd_boxPp_hovbox0Ppcmd_gluePpcmd_tagconstr.variablePpcmd_stringn_headsPpcmd_tagconstr.notationPpcmd_string;Ppcmd_print_break10Ppcmd_tagconstr.variablePpcmd_stringd_headPpcmd_tagconstr.notationPpcmd_string]Ppcmd_print_break10Ppcmd_tagconstr.variablePpcmd_stringAPpcmd_tagconstr.notationPpcmd_string *Ppcmd_print_break10Ppcmd_boxPp_hovbox2Ppcmd_gluePpcmd_tagconstr.variablePpcmd_stringtensorPpcmd_gluePpcmd_gluePpcmd_print_break10Ppcmd_tagconstr.notationPpcmd_string[Ppcmd_tagconstr.variablePpcmd_stringd_modelPpcmd_tagconstr.notationPpcmd_string]Ppcmd_print_break10Ppcmd_tagconstr.variablePpcmd_stringAPpcmd_tagconstr.notationPpcmd_string *Ppcmd_print_break10Ppcmd_boxPp_vbox0Ppcmd_gluePpcmd_gluePpcmd_boxPp_hvbox0Ppcmd_gluePpcmd_gluePpcmd_tagconstr.keywordPpcmd_stringmatchPpcmd_print_break12Ppcmd_boxPp_hovbox0Ppcmd_boxPp_hovbox0Ppcmd_tagconstr.variablePpcmd_stringnormalization_typePpcmd_print_break10Ppcmd_tagconstr.keywordPpcmd_stringwithPpcmd_gluePpcmd_print_break10Ppcmd_boxPp_hovbox4Ppcmd_gluePpcmd_string| Ppcmd_boxPp_hovbox0Ppcmd_gluePpcmd_boxPp_hovbox0Ppcmd_gluePpcmd_tagconstr.variablePpcmd_stringSomePpcmd_gluePpcmd_print_break10Ppcmd_tagconstr.variablePpcmd_stringLNPpcmd_string =>Ppcmd_print_break10Ppcmd_boxPp_hovbox2Ppcmd_gluePpcmd_tagconstr.variablePpcmd_stringtensorPpcmd_gluePpcmd_gluePpcmd_print_break10Ppcmd_tagconstr.notationPpcmd_string[Ppcmd_tagconstr.variablePpcmd_stringd_modelPpcmd_tagconstr.notationPpcmd_string]Ppcmd_print_break10Ppcmd_tagconstr.variablePpcmd_stringAPpcmd_gluePpcmd_print_break10Ppcmd_boxPp_hovbox4Ppcmd_gluePpcmd_string| Ppcmd_boxPp_hovbox0Ppcmd_gluePpcmd_boxPp_hovbox0Ppcmd_tagconstr.variablePpcmd_stringNonePpcmd_string =>Ppcmd_print_break10Ppcmd_boxPp_hovbox2Ppcmd_gluePpcmd_tagconstr.variablePpcmd_stringwith_defaultPpcmd_gluePpcmd_gluePpcmd_print_break10Ppcmd_string"()"Ppcmd_print_break10Ppcmd_tagconstr.variablePpcmd_stringIPpcmd_print_break10Ppcmd_tagconstr.keywordPpcmd_stringendPpcmd_tagconstr.notationPpcmd_string *Ppcmd_print_break10Ppcmd_boxPp_vbox0Ppcmd_gluePpcmd_gluePpcmd_boxPp_hvbox0Ppcmd_gluePpcmd_gluePpcmd_tagconstr.keywordPpcmd_stringmatchPpcmd_print_break12Ppcmd_boxPp_hovbox0Ppcmd_boxPp_hovbox0Ppcmd_tagconstr.variablePpcmd_stringnormalization_typePpcmd_print_break10Ppcmd_tagconstr.keywordPpcmd_stringwithPpcmd_gluePpcmd_print_break10Ppcmd_boxPp_hovbox4Ppcmd_gluePpcmd_string| Ppcmd_boxPp_hovbox0Ppcmd_gluePpcmd_boxPp_hovbox0Ppcmd_gluePpcmd_tagconstr.variablePpcmd_stringSomePpcmd_gluePpcmd_print_break10Ppcmd_tagconstr.variablePpcmd_stringLNPpcmd_string =>Ppcmd_print_break10Ppcmd_boxPp_hovbox2Ppcmd_gluePpcmd_tagconstr.variablePpcmd_stringtensorPpcmd_gluePpcmd_gluePpcmd_print_break10Ppcmd_tagconstr.notationPpcmd_string[Ppcmd_tagconstr.variablePpcmd_stringd_modelPpcmd_tagconstr.notationPpcmd_string]Ppcmd_print_break10Ppcmd_tagconstr.variablePpcmd_stringAPpcmd_gluePpcmd_print_break10Ppcmd_boxPp_hovbox4Ppcmd_gluePpcmd_string| Ppcmd_boxPp_hovbox0Ppcmd_gluePpcmd_boxPp_hovbox0Ppcmd_tagconstr.variablePpcmd_stringNonePpcmd_string =>Ppcmd_print_break10Ppcmd_boxPp_hovbox2Ppcmd_gluePpcmd_tagconstr.variablePpcmd_stringwith_defaultPpcmd_gluePpcmd_gluePpcmd_print_break10Ppcmd_string"()"Ppcmd_print_break10Ppcmd_tagconstr.variablePpcmd_stringIPpcmd_print_break10Ppcmd_tagconstr.keywordPpcmd_stringendPpcmd_string)
Ppcmd_boxPp_hovbox2Ppcmd_gluePpcmd_tagconstr.variablePpcmd_stringProperPpcmd_gluePpcmd_gluePpcmd_print_break10Ppcmd_boxPp_hovbox1Ppcmd_gluePpcmd_string(Ppcmd_boxPp_hovbox0Ppcmd_gluePpcmd_tagconstr.variablePpcmd_stringeqPpcmd_tagconstr.notationPpcmd_string ==>Ppcmd_print_break10Ppcmd_boxPp_hovbox0Ppcmd_gluePpcmd_gluePpcmd_gluePpcmd_tagconstr.pathPpcmd_stringTensorPpcmd_string.Ppcmd_tagconstr.referencePpcmd_stringeqfPpcmd_tagconstr.notationPpcmd_string ==>Ppcmd_print_break10Ppcmd_boxPp_hovbox2Ppcmd_gluePpcmd_tagconstr.variablePpcmd_stringoption_eqPpcmd_gluePpcmd_print_break10Ppcmd_gluePpcmd_gluePpcmd_tagconstr.pathPpcmd_stringTensorPpcmd_string.Ppcmd_tagconstr.referencePpcmd_stringeqfPpcmd_string)Ppcmd_print_break10Ppcmd_boxPp_hovbox1Ppcmd_gluePpcmd_string(Ppcmd_boxPp_hovbox2Ppcmd_gluePpcmd_gluePpcmd_gluePpcmd_tagconstr.pathPpcmd_stringHookedTransformerPpcmd_string.Ppcmd_gluePpcmd_tagconstr.pathPpcmd_stringHookedTransformerPpcmd_string.Ppcmd_tagconstr.referencePpcmd_stringattn_patternPpcmd_gluePpcmd_gluePpcmd_print_break10Ppcmd_tagconstr.variablePpcmd_stringuse_checkpointPpcmd_gluePpcmd_print_break10Ppcmd_tagconstr.variablePpcmd_stringW_EPpcmd_gluePpcmd_print_break10Ppcmd_tagconstr.variablePpcmd_stringW_posPpcmd_print_break10Ppcmd_tagconstr.variablePpcmd_stringblocks_paramsPpcmd_string)
```
![image](https://github.com/coq-community/vscoq/assets/396076/7e2776c9-cd8b-4f74-b872-786033301642)

```
$ coqc -v
The Coq Proof Assistant, version 8.19+alpha
compiled with OCaml 4.14.1
$ echo | coqtop
Welcome to Coq JasonGross-X1:/home/jgross/Downloads/coq/coq-master/_build/default,master (61ee398ed32f9334dd664ea8ed2697178e6e3844)

Coq < Coq <
```
Manually built the language server from 64af6a3e955e487787bff204845166eb9467a71b (is there a better way to get the version?)

**Comment by JasonGross** [comment]
Yep, that fixes it, thanks!  Can this sort of version mismatch be detected and warned about from the extension?

**Comment by rtetley** [comment]
> Yep, that fixes it, thanks! Can this sort of version mismatch be detected and warned about from the extension?

The mechanics to do this this are in place, just need to wire it now that we have different versions !

**Comment by maximedenes** [comment]
I implemented it, actually, in #568. Thanks for reporting!
